tests/projects/Sample_ConsoleApp_net7/Program.fs

```diff
@@ -0,0 +1,10 @@
+﻿// SDK version 7.0.100-preview.6 or newer has to be installed for this to work
+open System.Numerics
+
+[<Measure>] type potato
+
+let f (x: 'T when IMultiplyOperators<'T,'T,'T>) = x 
+
+// f 7.0<potato> 
+
+printfn $"Hello from F# {f 7.0}!"
```

Comment:
This is good.  Please flesh it out by porting a page or so of sample code from the various blog posts for .NET 7 generic math and the RFC

Reply:
I'll do that + some testing of the vsix